### PR TITLE
Add HECSS package

### PR DESCRIPTION
**Pull request by jochym**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/hecss/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+    - hecss_sampler --help
+
+about:
+  home: https://github.com/simplejson/simplejson
+  license: GPL-3.0-or-later
+  license_family: GPL
```
Comment (dopplershift) [inline recipes/hecss/meta.yaml:42]:
```suggestion
  license_family: GPL3
```

File: recipes/hecss/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - hecss
+    - hecss.core
+  requires:
+    - pip
+  commands:
+    - pip check
+    - hecss_sampler --help
+
+about:
+  home: https://github.com/simplejson/simplejson
```
Comment (dopplershift) [inline recipes/hecss/meta.yaml:40]:
```suggestion
  home: https://jochym.gitlab.io/hecss
``?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor tweaks and this is good.

File: recipes/hecss/meta.yaml (hunk 2 above)
Comment (jochym) [reply recipes/hecss/meta.yaml]: Thanks for the review. Fixed.

**Comment by jochym** [comment]
@dopplershift Thanks for the review. Changes committed. Can this be merged now?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**